### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/help-fastapi.md`

### PR DESCRIPTION
**Pull request by kty4119**
* Add Korean translation for `docs/ko/docs/help-fastapi.md`
* Related [FEATURE] Korean translations [FEATURE] Korean translations [FEATURE] Korean translations #2017

### CODE REVIEW

File: docs/ko/docs/help-fastapi.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+* # FastAPI 지원 - 도움말 받기
+
+  **FastAPI** 가 마음에 드시나요?
+
+  FastAPI, 다른 사용자, 개발자를 응원하고 싶으신가요?
+
+  혹은 **FastAPI** 에 대해 도움이 필요하신가요?
+
+  아주 간단하게 응원할 수 있습니다 (몇 번의 클릭만으로).
```
Comment (DevDae) [inline docs/ko/docs/help-fastapi.md:9]:
```suggestion
  (몇 번의 클릭만으로) 아주 간단하게 응원할 수 있습니다 .
```
I think that if the expression "몇 번의 클릭만으로" comes in front of the sentence, it is easier to understand in Korean!
feel free to my opinion! thanks 🚀

File: docs/ko/docs/help-fastapi.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+  또한 도움을 받을 수 있는 방법도 몇 가지 있습니다.
+
+  ## 뉴스레터 구독
+
+  [**FastAPI와 친구** 뉴스레터](https://github.com/tiangolo/fastapi/blob/master/newsletter)를 구독하여 최신 정보를 유지할 수 있습니다{.internal-link target=_blank}:
```
Comment (DevDae) [inline docs/ko/docs/help-fastapi.md:15]:
```suggestion
  [**FastAPI와 친구** 뉴스레터](https://github.com/tiangolo/fastapi/blob/master/newsletter)를 구독하여 최신 정보를 얻을 수 있습니다{.internal-link target=_blank}:
```
I think "얻다" can replace the word "유지하다". 😄

File: docs/ko/docs/help-fastapi.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+  여러분은 할 수 있습니다:
+
+  - [**GitHub**에서 팔로우하기](https://github.com/tiangolo).
+  - 당신에게 도움이 될 저의 다른 오픈소스 프로젝트를 확인하십시오.
+    - 새로운 오픈소스 프로젝트를 만들었을 때 확인하려면 팔로우 하십시오.
+
+  - [**Twitter**에서 팔로우하기](https://twitter.com/tiangolo).
+    - FastAPI의 사용 용도를 알려주세요 (그것을 듣는 것을 좋아합니다).
+    - 발표 또는 새로운 툴 출시할 때 들으십시오.
```
Comment (DevDae) [inline docs/ko/docs/help-fastapi.md:53]: I can't understand what "툴" 's mean 😭 
please give your opinion! thanks

File: docs/ko/docs/help-fastapi.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+  👥 [디스코드 채팅 서버](https://discord.gg/VQjSZaeJmf) 👥 에 가입하고 FastAPI 커뮤니티에서 다른 사람들과 어울리세요.
+
+  !!! tip 질문이 있는 경우, [GitHub 이슈 ](https://github.com/tiangolo/fastapi/issues/new/choose) 에서 질문하십시오, [FastAPI 전문가](https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/fastapi-people.md#experts) 의 도움을 받을 가능성이 높습니다{.internal-link target=_blank} .
+
+  ```
+  다른 일반적인 대화에서만 채팅을 사용하십시오.
+  ```
+
+  기존 [지터 채팅](https://gitter.im/tiangolo/fastapi) 이 있지만 채널과 고급기능이 없어서 대화를 하기가 조금 어렵기 때문에 지금은 디스코드가 권장되는 시스템입니다.
```
Comment (DevDae) [inline docs/ko/docs/help-fastapi.md:127]:
```suggestion
  기존 [지터 채팅](https://gitter.im/tiangolo/fastapi) 이 있지만 채널과 고급기능이 없어서 대화를 하기가 조금 어렵기 때문에 지금은 디스코드가 권장되고 있습니다.
```

File: docs/ko/docs/help-fastapi.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+
+  ### 질문을 위해 채팅을 사용하지 마십시오
+
+  채팅은 더 많은 "자유로운 대화"를 허용하기 때문에, 너무 일반적인 질문이나 대답하기 어려운 질문을 쉽게 질문을 할 수 있으므로, 답변을 받지 못할 수 있습니다.
+
+  GitHub 이슈에서의 템플릿은 올바른 질문을 작성하도록 안내하여 더 쉽게 좋은 답변을 얻거나 질문하기 전에 스스로 문제를 해결할 수도 있습니다. 그리고 GitHub에서는 시간이 조금 걸리더라도 항상 모든 것에 답할 수 있습니다. 채팅 시스템에서는 개인적으로 그렇게 할 수 없습니다. 😅
```
Comment (DevDae) [inline docs/ko/docs/help-fastapi.md:133]:
```suggestion
  GitHub 이슈에서의 템플릿은 올바른 질문을 작성하도록 안내하여 더 쉽게 좋은 답변을 얻거나 질문하기 전에 스스로 문제를 해결할 수도 있습니다. 그리고 GitHub에서는 시간이 조금 걸리더라도 항상 모든 것에 답을 얻을 수 있습니다. 채팅 시스템에서는 개인적으로 그렇게 할 수 없습니다. 😅
```

**Review by DevDae — CHANGES_REQUESTED** [review]
Appreciate your hard works! 🙏 
I will wait your comments, thank you!

**Comment by tiangolo** [comment]
📝 Docs preview for commit b033d6471a7c2bee9b89adc8569a1057332a161a at: https://6499e34684c0620543c16807--fastapi.netlify.app

**Comment by tiangolo** [comment]
Thank you! I'm still waiting for others to review this to be able to merge it, meanwhile, maybe you could help me review some of the other Korean translations, that way if they are approved by each other, I could merge them. :nerd_face: 

https://github.com/tiangolo/fastapi/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+label%3Alang-ko+label%3Aawaiting-review

Maybe @DevDae could review it again after the requested changes? :pray:

**Review by junah201 — APPROVED** [review]

**Comment by alejsdev** [comment]
Thanks for the help! @kty4119 :nerd_face: :100: 

And thanks for the reviews @DevDae @junah201 :mag:

**Pull request merged**